inductiva/resources/machines.py

```diff
@@ -58,6 +59,9 @@ def from_api_response(cls, resp: dict):
         machine_group.register = False
         return machine_group
 
+    def __str__(self):
+        return f"Machine Group: {self.name}"
```

Comment:
I don't see where the class has the attribute `name`. Also, maybe:

```python
return f"Machine Group with {self.num_machines}"
```

Reply:
The `name` is within the base class, and is set when calling the `register()`. Will add that!